README.md

```diff
@@ -1 +1,51 @@
-# Cadastro-central
+# Central register 🇺🇸
+
+## About
+Central register is composed by REST APIs, `Client`, `Address`, `IdentityDocument` and `Telephone`. They are organized as shown in the following image, and its purpose is to register people, as `Client`, by the attributes: `Name`, `MaritalStatus`, list of `IdentityDocument`, list of `Telephone` and `Address`.
+
+[![Central Register Schema](https://i.postimg.cc/tThGsKDR/central-register-schema.png)](https://postimg.cc/625mS1DF)
+## First steps
+### Database
+Before running our application, you have to create a postgres database locally with the following attributes:
+#### Server
+* Host: `localhost`
+* Port: `5432`
+* Database name: `central_register`
+#### Authentication
+* Authentication: `Database Native`
+* Username: `postgres`
+* Password: `postgres`
+
+Also, we used the [Lombok library](https://projectlombok.org/) in the project, so if you open it in an IDE, like IntelliJ, you should install a plugin to analyze it better.
```

Comment:
I think here maybe it's better 'to run the application without errors.' intead of 'to analyze it better.'